vms/spdagvm/vm.go

```diff
@@ -328,10 +330,19 @@ func (vm *VM) Send(amount uint64, assetID, toAddrStr string, fromPKs []string) (
 		if err != nil {
 			return "", err
 		}
-		// Parse the crpyo.PrivateKey repr. to a crypto.PrivateKeySECP256K1R
+		// Parse the crypto.PrivateKey repr. to a crypto.PrivateKeySECP256K1R
 		keychain.Add(pk.(*crypto.PrivateKeySECP256K1R))
 	}
 
+	toAddrBal, err := vm.GetBalance(toAddrStr, "")
```

Comment:
I'm not sure that this should be an error. It is possible, if the amount of money in the total UTXO set is > MathUint64, for there to be an address that controls > MathUint64. However, the transaction format limits MathUint64 to be sent in any transaction. Preventing the transactions from being created in the client doesn't prevent these transactions from being issued and accepted.